apps/hash-integration-worker/src/mappings.ts

```diff
@@ -22,6 +22,13 @@ export type PartialEntity = {
   entityTypeId: VersionedUrl;
 };
 
+const toIsoString = (date: string | Date): string => {
+  if (typeof date === "string") {
+    return date;
+  }
+  return date.toISOString();
+};
```

Comment:
You could save the tenaries below by updating it to this
```suggestion
const toIsoStringIfDefined = (date?: string | Date): string => {
  if (!date || typeof date === "string") {
    return date;
  }
  return date.toISOString();
};
```

Reply:
We agreed offline that we don't want to do this as it's weird to pass in a maybe-undefined value just to get another undefined back out.